.php_cs

```diff
@@ -0,0 +1,80 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->in(__DIR__);
+
+$fixers = array(
```

Comment:
Sorry I haven't checked myself yet, but does this custom list cover PSR2? I just would like Yasumi to be PSR2 compliant. Any additional ones are fine to me, but not mandatory.